tutorial.md

```diff
@@ -1,53 +1,80 @@
 ---
-title: Bndtools Tutorial
+Title: Bndtools Tutorial
```

Comment:
I think this should be as it was before `title`.

Reply:
@isalvadori Do you plan to update this PR?

Reply:
@bjhargrave  Yes, sorry for the delay. I will update it today!